builtin/providers/azurerm/resource_arm_virtual_machine.go

```diff
@@ -95,21 +95,28 @@ func resourceArmVirtualMachine() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"image_id": {
+							Type:          schema.TypeString,
+							Optional:      true,
+							ForceNew:      true,
+							ConflictsWith: []string{"storage_image_reference.publisher", "storage_image_reference.offer", "storage_image_reference.sku"},
```

Comment:
Given `storage_image_reference` is a set, I believe this currently won't work? As such can we remove this line?